code/modules/projectiles/ammo_datums.dm

```diff
@@ -2594,11 +2598,10 @@ datum/ammo/bullet/revolver/tp44
 /datum/ammo/flamethrower/blue
 	name = "blue flame"
 	hud_state = "flame_blue"
-	max_range = 6
+	max_range = 7
 	fire_color = "blue"
 	burnlevel = 46
 	burntime = 40
-	fire_delay = 20
```

Comment:
Ditto

Reply:
This is just cleanup. Flamer ammo has a this var uniquely (line 2575 above) which is applied to the flamer on reload, but all flamer ammo has an identical delay to it and it's applied in a weird way (on reload, the gun's fire delay is set to the ammo datum's initial, then set back to the gun's initial on unload). I adjusted the flamer's default fire_delay to the value applied by the ammo, since that's what it was always.

Reply:
The exception to this is the miniflamer - it has a coded fire_delay of 2.5s, but since loading ammo sets it anyway this has never done anything before.